BusinessLogic/Services/EmailService.cs

```diff
@@ -13,20 +13,20 @@ public EmailService(IEmailProvider provider)
         this.provider = provider;
     }
 
-    public void SendEmail(string reciever, TestResultProjection testResult)
+    public void SendEmail(string reciever, Guid testResultId)
```

Comment:
testResultId word niet gebruikt?

Reply:
nee klopt is voor c-005, ik moest iets doorgeven want anders build ie niet....